src/foam/lib/json/AnyParser.java

```diff
@@ -18,6 +18,6 @@ public AnyParser() {
         new BooleanParser(),
         new LongParser(),
         new DoubleParser(),
-        new FObjectParser()));
+        new ExprParser()));
```

Comment:
Yes, except L:21 is already an AltParser, so you don't need another one. Just call it PropertyParser.